CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/DatabaseRequestHandler.java

```diff
@@ -187,10 +187,16 @@ public void saveDocuments(Args args) throws CouchbaseLiteException {
         });
     }
 
-    public void save(Args args) throws CouchbaseLiteException {
+    public String save(Args args) throws CouchbaseLiteException {
         Database database = args.get("database");
         MutableDocument document = args.get("document");
-        database.save(document);
+        try {
+            database.save(document);
+            return "Document Saved";
+        } catch (CouchbaseLiteException e) {
```

Comment:
I think this will cause issues. have you rebuilt with this branch and run a full regression? 
public String save(Args args) throws CouchbaseLiteException indicate whenever this is a CouchbaseLiteException, it will throw. in your change, you catch it, and return a message, therefore the exception will be hidden. I think python side will need to deal with this exception. 

Actually if you check the swift code, requesthandler defines throw when exception occurs.

Reply:
build the Test app and have a run to make sure it works

Reply:
Eunice, It is correct . When tested with ios without try I was not getting any error message . With this change on IOS app, I am getting errors that are raised by the request handlers.